api_tests/src/actors/actor.ts

```diff
@@ -664,12 +664,10 @@ export class Actor {
     }
 
     public async assertAlphaNotDeployed() {
-        await sleep(3000); // It is meaningless to assert before cnd processes a new block
```

Comment:
Can you explain why this is not valid anymore please?

Reply:
Please see commit message: https://github.com/comit-network/comit-rs/pull/2477/commits/943af8ba3469e5607308761308e6156cd3ba6032

> Stabilize e2e by reducing sleep time
> 
> The sleep was done up to 4*3s in one test: Each time a
`assert*NotDeployed` is called.
> 
> This causes failure as it was toying with the timeout line.

Reply:
Thanks for pointing this out. Nevertheles, the comment said: 
> It is meaningless to assert before cnd processes a new block

Does this not apply anymore or has this been resolved otherwise?